deeplearning/ml4pl/models/ggnn/run.py

```diff
@@ -353,7 +353,9 @@ def data2input(self, batch):
         # maybe add labels
         if batch.y is not None:
             inputs.update({
-                "labels": batch.y,
+                # TODO this is a hotfix and not required after re-collating the data_train.pb
+                # of the rebuttal-classifyapp-dataset
+                "labels": batch.y - 1,
```

Comment:
Is this duplicated with -1 offset added to `nx2data()` below?

Reply:
yes, but it was working if you still had the old preprocessing cached. I un-hotfixed this line below and reprocessed my data. should all work now (since yesterday evening).

Reply:
Great! If the results are looking good, let's merge and I'll submit a new PR for devmap